src/objects/jira_base.py

```diff
@@ -5,8 +5,11 @@
 from jira import Issue
 from jira import JIRA
 from jira.exceptions import JIRAError
+from pyhelper_utils.general import ignore_exceptions
 from simple_logger.logger import get_logger
 
+LOGGER = get_logger(name=__name__)
+
```

Comment:
```suggestion
LOGGER = get_logger(name=__name__)
```

Reply:
Delete extra line?

Reply:
This is due to linting. There should be two blank lines between a class and anything else.